services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+# Virtual Machines Taxonomy
+
+This _service-level taxonomy_ documents the minimal set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+|	CCC-030101	|	Instance Types	|	Offers a range of instance types with different specifications for CPU, GPU, memory, storage optimisation and networking capacity.	|
+|	CCC-030102	|	Operating System Options 	|	A selection of operating systems for virtual machine instances.	|
+|	CCC-030103	|	Ephemeral Storage	|	Temporary storage available to the VM which is lost when the instance is stopped or terminated	|
+|	CCC-030104	|	Availability	|	Ensuring high availability of virtual machine instances through redundancy and multiple (availability) zones within a region.	|
+|	CCC-030105	|	Identity and Access Management	|	Implementing identity and access management features such as key pairs, JIT and MFA to control user access to virtual machine instances.	|
+|	CCC-030106	|	Monitoring and Logging	|	Offering monitoring and logging capabilities to track performance metrics, user access and security events.	|
+|	CCC-030107	|	Backup and Restore	|	Providing backup and disaster recovery solutions for virtual machine instances and associated data, including snapshot-based backups, incremental backups, and point-in-time recovery.	|
+|	CCC-030108	|	Encryption at Rest	|	Encrypting data stored by virtual machine instances to protect against unauthorized access.	|
+|	CCC-030109	|	Encryption in Transit	|	Encrypting data transmitted to and from virtual machine instances to protect against interception.	|
```

Comment:
What does VM do to enable encryption on transit?

Reply:
I put this in early on because obviously encryption=essential, but I agree that it's not really the VM doing this part. I think I'll remove this one.